.github/workflows/docker.yaml

```diff
@@ -0,0 +1,54 @@
+name: Push Docker images to DockerHub and ECR
+
+on:
+  push:
+    branches: [main, master]
```

Comment:
Make this just `main` (@ErikSchierboom Can we do this everywhere - atm someone could push a new branch called `master` and this would run on it, which seems bad)